process/slack_etiquette.md

```diff
@@ -0,0 +1,64 @@
+
+### :star2: CodeBuddies is asynchronous & international
+We have members on every continent (except Antartica), and many of us are working professionals. You may get help within 5 minutes if somebody who knows the answer is online looking for something to do -- but it’s entirely possible that you won’t get a response for an hour or day or more. If you’re asking about a less popular tool, or a specific edge case, you may even get an answer _**several days later.**_
+.
+So remember to keep CodeBuddies open in the background for a while after asking a question - or check back periodically.
+.
+Please don't cross post the same question in multiple channels. People in this community may not be super-chatty at any given moment, but we are reading posts across all channels.  Someone will eventually see (and help) you. _**We promise**_.
+
+### :star2: Some channels are more active than others
+Questions in #general or #advice tend to get answered quickly, or they get pushed off the screen & forgotten. We have a handful of dedicated people that read the scrollback and try to answer forgotten questions, but there’s a lot of discussion/movement happening, you may want to re-post after a few days.  
+.
+Conversely, if a channel is particularly quiet, please do not post "this channel is quiet".  Instead, try posting a question/article/or discussion point that others in the channel might comment on or enjoy.
+
+### :star2: Don’t “ask to ask”
+Newcomers very commonly ask about asking a question in one of a few ways.
+.
+_**“Does anyone know about [X tool or library]?”**_
+The answer is probably **“yes,”** but nobody can help you until you state what your problem **is**. All too often by the time somebody asks what the **real** question is, the person who needed help has disconnected. 
+.
+_**Please be specific with your questions so all of us can help you even when you aren't online.**_ Share a code snippet, a link to where you're seeing the issue, a description of what is confusing you, and specifics on what you've tried so far.  More often than not, it also helps to paste your error logs into a snippet or text file (instructions here on how to do that in Slack) to accompany your question/issue.
+.
+_**“Is it okay if I ask about [X tool or library] here?”**_
+We have a dizzying array of channels, which we’re aware of and for which we apologize. We have utility channels such as #announcements, #general, #advice, etc.  All our channels are listed alphabetically. 
+.
+Instead of asking, browse the Channels list for any that seem like obvious places to ask your question. If you don’t see any topics or Channels that fit, don’t hesitate to post it in #general or #advice. If there’s a better place for it, one of the moderators or other helpful users will suggest it, but don't worry - off topic messages aren’t going to get you banned.
+
+### :star2: Don’t use `@here`, `@channel`, or ping / DM members unless it's directly relevant
```

Comment:
I think this might be better reworded to make sure the key point is first. "We are all in different timezones and be careful when DMing or using @-username, @here, or @channel." I'm really just looking for the first line to be the important thing.